Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -115,80 +115,158 @@ public void Roslyn_Settings_ValidSetup()
 
             AssertWarningsAreNotTreatedAsErrorsNorIgnored(result);
         }
-
+        
         [TestMethod]
-        [Description("Checks the happy path i.e. settings exist in config, no reason to exclude the project")]
```

Comment:
The previous description was cut-and-paste from another test and didn't relate to the actual test. I've also updated the test name to make it more accurate.